tests/org.wso2.carbon.configuration.test/src/test/resources/META-INF/services/javax.annotation.processing.Processor

```diff
@@ -0,0 +1 @@
+org.wso2.carbon.configuration.ConfigurationProcessor
```

Comment:
we don't need to add service provider for the testcase. It should come from the configuration dependency